lib/lte_link_control/lte_lc.c

```diff
@@ -222,7 +222,8 @@ static void at_handler_cereg(const char *response)
 	}
 
 	/* Cell update event */
-	if (memcmp(&cell, &prev_cell, sizeof(struct lte_lc_cell))) {
+	if ((cell.id != prev_cell.id) || (cell.tac != prev_cell.tac) ||
+	    (cell.mcc != prev_cell.mcc) || (cell.mnc != prev_cell.mnc)) {
```

Comment:
What about `phys_cell_id`?

Reply:
Yep, added that too. Also updated the changelog